pkg/pushsync/pushsync.go

```diff
@@ -213,19 +213,18 @@ func (ps *PushSync) handler(ctx context.Context, p p2p.Peer, stream p2p.Stream)
 
 	// forwarding replication
 	storedChunk := false
-	if ps.topologyDriver.IsWithinDepth(chunkAddress) {
+	if ps.warmedUp() && ps.topologyDriver.IsWithinDepth(chunkAddress) {
```

Comment:
where did the `warmedUp` come from? it is used to mitigate sending back receipts when the node has not yet warmed up and I think it is not relevant in this context

Reply:
checking depth depends on the kademlia topology of the node. If it has not warmed up, the node may accept chunks that are not normally in it's neighborhood.